audformat/core/common.py

```diff
@@ -227,13 +227,22 @@ class DefineBase:
 
     @classmethod
     def assert_has_value(cls, value):
```

Comment:
This one we could rename to `assert_class_has_attribute`?

Reply:
I renamed it to `assert_has_attribute_value`